lib/ts/recipe/session/recipe.ts

```diff
@@ -27,6 +30,25 @@ export default class Recipe extends RecipeModule<unknown, any> {
 
         SuperTokensWebsite.init({
             ...config,
+            override: {
+                functions: (originalImpl, builder) => {
+                    builder!.override((oI) => ({
+                        ...oI,
+                        getGlobalClaimValidators: (input) => {
```

Comment:
should we use arrow functions here? I think there was an issue with them, and users had to use non arrow functions for override? Or vice versa. Do you rememebr what i am talking about?

Reply:
yup, there is even a PR checklist item in st-node for this. It doesn't matter here, since we are not using `this`, but I'll update it anyway for consistency.